tests/test-utils.php

```diff
@@ -810,4 +810,22 @@ public function testReplacePathConstsAddSlashes() {
 		$actual   = Utils\replace_path_consts( $source, "C:\Users\\test's\site" );
 		$this->assertSame( $expected, $actual );
 	}
+
+	/** @dataProvider dataValidClassAndMethodPair */
+	public function testValidClassAndMethodPair( $pair, $is_valid ) {
+		$this->assertEquals( $pair, Utils\is_valid_class_and_method_pair( $is_valid ) );
```

Comment:
The arguments seem to be mixed up here.
```suggestion
		$this->assertEquals( $is_valid, Utils\is_valid_class_and_method_pair( $pair ) );
```

Reply:
🤦 Ack, sorry, thanks for correcting that!